api/proto/metrics/metrics.proto

```diff
@@ -236,7 +221,7 @@ message deviceMetric {
   // inactive in the controller. 'dormantTimeInSeconds' is subjected to change based
   // of network connectivity.
   uint64 dormant_time_in_seconds = 22;
-  repeated StorageMetric storage_metrics = 23;
+  // Depreated tag 23
```

Comment:
I think it makes sense to keep this even if we currently do not fill in the StorageMetrics.